swatch-core/src/main/java/org/candlepin/subscriptions/db/model/HardwareMeasurementType.java

```diff
@@ -34,25 +34,46 @@ public enum HardwareMeasurementType {
   // integration
   @Deprecated
   AWS_CLOUDIGRADE,
-  GOOGLE,
+  GOOGLE("GCP"),
```

Comment:
Since `HardwareMeasurementType` is used in some entities like `Host`, if we rename `GOOGLE` with `GCP`, we would need to migrate the existing hosts. Moreover, for events, we do receive events with `Google` (see Event.CloudProvider`.

Therefore, I decided to instead of changing GOOGLE with GCP, use the concept of aliases. So, if `google` or `gcp` is used, it will be mapped as CLOUD.